README.md

```diff
@@ -471,17 +437,41 @@ You can also join our discord server
 			.
 			}
     ```
+	* [Sample 1](https://www.dropbox.com/s/ukev1hi4kguomri/100.gv?dl=0) (100 Vertices , 11KB)
+	* [Sample 2](https://www.dropbox.com/s/vpvvliz96mdea1p/1000.gv?dl=0) (1000 Vertices , 106KB)
 	* [Online Visualization](https://dreampuf.github.io/GraphvizOnline/)
 
 - [Pickle(.p)](https://docs.python.org/3.5/library/pickle.html) (Binary Format)	
- 			
+	* [Sample 1](https://www.dropbox.com/s/4s8zt9i13z39gts/100.p?dl=0) (100 Vertices , 12KB)
+ 	* [Sample 2](https://www.dropbox.com/s/fzurqu5au0p1b54/1000.p?dl=0) (1000 Vertices , 340KB)		
+
+
+## Example of Usage
+
+- Generate synthetic data for graph processing frameworks (some of them mentioned here) performance-analysis 			 
+	- [Medusa](https://github.com/JianlongZhong/Medusa "Medusa") 
+	- [Totem](https://github.com/netsyslab/Totem "Totem")
+	- [Frog](https://github.com/AndrewStallman/Frog "Frog")
+	- [CuSha](https://github.com/farkhor/CuSha "CuSha")
```

Comment:
Are these links correct?
I've never seen this format `[link_name](link "name")`

Reply:
Great!